packages/react/src/analytics/integrations/Vitorino/Vitorino.ts

```diff
@@ -282,41 +160,25 @@ export default class Vitorino extends integrations.Integration<VitorinoIntegrati
    * @returns Promise that will resolve when the method finishes.
    */
   override async track(data: EventData<TrackTypesValues>) {
-    await this.initializePromise;
-    await this.userIdPromise;
-
-    try {
-      const PageTypes = this.getPageTypeFromEvent(data.event);
-      let safePageTypes;
+    await this.forter?.track(data);
+    await this.riskified?.track(data);
+  }
 
-      if (isArray(PageTypes)) {
-        safePageTypes = PageTypes.length ? PageTypes : [undefined];
-      } else {
-        safePageTypes = [PageTypes];
-      }
+  getForter() {
+    return this.forter;
+  }
 
-      safePageTypes.forEach(pageType => this.callVitorino(pageType));
-    } catch (error) {
-      utils.logger.error(`${VITORINO_CALL_ERROR_MESSAGE} ${error}`);
-    }
+  getRiskified() {
+    return this.riskified;
   }
 
-  /**
-   * Calls the callback function given from the config call to register a page view
-   * change.
-   *
-   * @param page - The `page` parameter of the event.
-   */
-  callVitorino(page?: string) {
-    if (this.isVitorinoConfigured && this.vitorinoTrackCallback) {
-      this.vitorinoTrackCallback(page);
-    }
+  getBaseOptions() {
```

Comment:
Is this necessary to be exposed as well?

Reply:
Removed